examples/wrapper-types/uniffi.toml

```diff
@@ -9,3 +9,30 @@ cdylib_name = "wrapper_types"
 
 [bindings.kotlin]
 cdylib_name = "wrapper_types"
+
+# Python wrapper implementation for JsonObject
+[wrappers.JsonObject.python]
+
+code = """
+class JsonWrapper:
+    fn from_wrapped(string):
```

Comment:
`from_wrapped/into_wrapped` is my latest attempt at making it clear which direction these functions are going.  I think `wrap()` and `unwrap()` would do just fine as well though.

Reply:
huh - I really think we need to come up with something better here, because my brain hurts every time I think about it. It says "a jsonobject is a wrapper around a string", or shorter, "a json object wraps a string" - which is reflected in the `JsonWrapper` name. So in this world, "wrap" would take a string and return a JSONObject (ie, take the thing being wrapped, and return the wrapper). However, 'wrapped' seems more problematic - after you wrap a string, it is wrapped in a `JsonWrapper` - so `from_wrapped` could just as easily mean "from json wrapper" - which is the opposite of what's going on - although your example *also* looks correct if you tilt your head.

`from_wrapper` is more consistent in this example, given we've named the class `JsonWrapper` - but that's the exact opposite of your intent. `from_primitive/into_primitive` seem to avoid that.